misk-testing/src/main/kotlin/misk/web/WebTestingModule.kt

```diff
@@ -11,12 +11,27 @@ import misk.security.ssl.SslLoader
  * A module that starts an embedded Jetty web server configured for testing. The server supports
  * both plaintext and TLS.
  */
+@Deprecated(
+  message = "This modules incorrectly imports MiskTestingServiceModule",
```

Comment:
Better to say to "install both WebServerTestingModule and MiskTestingServiceModule instead" but also should have more reasoning here than that just that it's incorrect.